scripts/swadeNpcImporter.js

```diff
@@ -29,11 +29,8 @@ Hooks.on('ready', async () => {
 
 Hooks.on('renderActorDirectory', async (app, html, data) => {
   if (game.users.get(game.userId).can('ACTOR_CREATE') == true) {
-    const npcImporterButton = $(
-      `<button style="margin: 4px; padding: 1px 6px;"><i class="fas fa-file-import"> ${game.i18n.localize(
-        'npcImporter.HTML.ActorImporter'
-      )}</i></button>`
-    );
+    const npcImporterButton = `<button style="width: calc(100% - 8px);"><i class="fas fa-align-left"></i>${game.i18n.localize('npcImporter.HTML.StatBlockImporterTitle')}</button>`;
```

Comment:
need to wrap this as a DOM element, otherwise, click doesn't work :/

```js
const npcImporterButton = $(
      `<button style="width: calc(100% - 8px);"><i class="fas fa-align-left"></i>${game.i18n.localize(
        'npcImporter.HTML.StatBlockImporterTitle'
      )}</button>`
    );
```

Reply:
Will do.

Reply:
Done.